src/any_agent/frameworks/any_agent.py

```diff
@@ -49,6 +50,8 @@ def __init__(
         self._mcp_servers: list[_MCPServerBase[Any]] = []
         self._main_agent_tools: list[Any] = []
 
+        self.server: A2AServer
```

Comment:
Not sure I follow what this line is doing, inside `serve` the stuff gets stored in `self._server`

Reply:
I'll check it out. The A2AServer further wraps a Starlette App deployed on a uvicorn Server, so it could be one of those _or_ I may be getting the name wrong.